static/src/javascripts/projects/commercial/modules/dfp/dfp-api.spec.js

```
@@ -276,7 +276,7 @@ describe('DFP', () => {
     it('hides all ad slots when all DFP advertising is disabled', () => {
         commercialFeatures.dfpAdvertising = false;
 
-        return prepareGoogletag().then(() => {
```

You need to return the promises, otherwise the tests fail silently.